### PR TITLE
organisations settings

### LINKED ISSUE #2613: Emails: Add toggle for warning mails
**Is your feature request related to a problem? Please describe.**
We have added support in the backend to toggle on/off warning mails i.e. In addition to critical notification, an organization will get email for warnings as well

**Describe the solution you'd like**
Add UI for toggling this on/off

### PR DESCRIPTION
**Pull request by akanshaaa19**
## Summary
Added the following: 
- Toggle for warning mails
- Low balancethreshold
- critical balance threshold
closes #2613 #2612 #2703 

image

### CODE REVIEW

**Review by mdshamoon — CHANGES_REQUESTED** [review]
Overall looks good. Few changes that I think we should do-

- Send warning email checkbox should be the first item
- If checkbox is unchecked we should disable the input fields for low and critical balance threshold. (You can refer to flows section under settings)

File: src/containers/SettingList/Organisation/Organisation.test.tsx [hunk 1]
```diff
@@ -48,14 +67,49 @@ test('it renders component in edit mode', async () => {
   // loading is show initially
   expect(getByText('Loading...')).toBeInTheDocument();
 
+  //correct values are rendered in the form
   await waitFor(() => {
-    const phoneNumber = getByTestId('phoneNumber');
+    const inputElements = screen.getAllByRole('textbox');
+    const numberInputElements = screen.getAllByRole('spinbutton');
+
+    const orgName = inputElements[0] as unknown as HTMLInputElement;
```
Comment (mdshamoon) [inline src/containers/SettingList/Organisation/Organisation.test.tsx:75]:
```suggestion
    const orgName = inputElements[0] as HTMLInputElement;
```
Any reason we are converting it to unknown first?

File: src/containers/SettingList/Organisation/Organisation.test.tsx [hunk 2]
```diff
@@ -48,14 +67,49 @@ test('it renders component in edit mode', async () => {
...
+    fireEvent.change(inputElements[0], { target: { value: 'Glific' } });
+    fireEvent.change(inputElements[1], { target: { value: 'Please change me, NOW!' } });
+    fireEvent.change(numberInputElements[0], { target: { value: '10' } });
+    fireEvent.change(numberInputElements[1], { target: { value: '5' } });
+  });
+
+  await waitFor(() => {
+    const submitButton = screen.getByText('Save');
+    expect(submitButton).toBeInTheDocument();
+    fireEvent.click(submitButton);
```
Comment (mdshamoon) [inline src/containers/SettingList/Organisation/Organisation.test.tsx:113]: Events should always be outside of waitFor
and the end of a test should always be an assertion

**Review by mdshamoon — COMMENTED** [review]
Overall looks good. Added some comments, please check.

File: src/containers/SettingList/Organisation/Organisation.test.tsx (hunk 1 above)
Comment (akanshaaa19) [reply src/containers/SettingList/Organisation/Organisation.test.tsx:75]: because it is an HTMLElement so it doesn't have value prop

Comment (mdshamoon) [reply src/containers/SettingList/Organisation/Organisation.test.tsx:75]: But we can directly do `as HTMLInputElement`. Why do we need to add `as unknown`

**Review by mdshamoon — APPROVED** [review]

**Pull request merged**